pkg/generate/app/app_test.go

```diff
@@ -14,6 +14,7 @@ import (
 	imageapi "github.com/openshift/origin/pkg/image/api"
 
 	_ "github.com/openshift/origin/pkg/api/install"
+	"strings"
```

Comment:
I realized I left that in there last night and knew you'd call it out :smile: You're so predictable, @soltysh.  I should've left some long ifs in there too.